mlir/lib/Dialect/XeGPU/IR/XeGPUOps.cpp

```diff
@@ -112,6 +112,68 @@ isValidGatherScatterParams(Type maskTy, VectorType valueTy,
 //===----------------------------------------------------------------------===//
 // XeGPU_CreateNdDescOp
 //===----------------------------------------------------------------------===//
+
+void CreateNdDescOp::build(OpBuilder &builder, OperationState &state,
+                           Type tdesc, TypedValue<MemRefType> source) {
+  [[maybe_unused]] auto ty = source.getType();
+  assert(ty.hasStaticShape() && "expecting a memref with static shape");
+
+  build(builder, state, tdesc, source, ValueRange({}) /* dynamic offsets */,
+        ValueRange({}) /* empty dynamic shape */,
+        ValueRange({}) /* empty dynamic strides */,
+        builder.getDenseI64ArrayAttr({}) /* const offsets */,
```

Comment:
nit: I think you can just use `ValueRange{}` and `DenseI64ArrayAttr()` directly

Reply:
fixed